mp/src/game/shared/neo/neo_gamerules.h

```diff
@@ -182,7 +193,7 @@ class CNEORules : public CHL2MPRules, public CGameEventListener
 
 	virtual bool CheckGameOver(void) OVERRIDE;
 
-	float GetRoundRemainingTime() const;
+	float GetRoundRemainingTime();
```

Comment:
Why `GetRoundRemainingTime` lost the `const`?

Reply:
The value it was returning wasn't the correct one when switching between gamemodes with two different time limits

Reply:
most noticeable on the clients hud at the top

Reply:
That doesn't sound like it has what's related to const being dropped since the member variables aren't altered? The only thing I see is `GetGameType()` not being const due to the parent class, but in that case you can just use `m_nGameTypeSelected` instead.